code/game/objects/items/weapons/cards_ids.dm

```diff
@@ -93,6 +94,8 @@
 
 /obj/item/weapon/card/id/examine(mob/user)
 	..()
+	if(registered_race && registered_race.id != "human")

```

Comment:
(I was trying to hide my fuck ups if somehow an id was generated without a species)

I'll see what I can do.